Algorithms/Strings/NaiveStringSearch.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+namespace Algorithms.Strings
+{
+    /// <summary>
+    /// Implements the traditional naive string matching algorithm in C#.
+    /// Cost:  O(n*m), where m is pattern length and n is content length.
+    /// </summary>
+    class NaiveStringSearch
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=4766632)